### ISSUE TITLE
Create dataset loader for GlobalWoZ V2.0

### ISSUE THREAD

**Issue by SamuelCahyawijaya** [SEACrowd/seacrowd-datahub#83]
Dataloader name: `globalwoz_v2.0/globalwoz_v2.0.py`
DataCatalogue: http://seacrowd.github.io/seacrowd-catalogue/card.html?globalwoz_v2.0

|     Dataset| globalwoz_v2.0 |
|-------------|---|
| Description | This is the data of the paper “GlobalWoZ: Globalizing MultiWoZ to Develop Multilingual Task-Oriented Dialogue Systems” accepted by ACL 2022. The dataset contains several sub-datasets in 20 languages and 3 schemes (F&E, E&F, F&F), including Indonesian (id), Thai (th), and Vietnamese (vi) language. The method is based on translating dialogue templates and filling them with local entities in
the target language countries. |
| Subsets | E&F_id, E&F_th, E&F_vi, F&E_id, F&E_th, F&E_vi, F&F_id, F&F_th, F&F_vi |
| Languages | ind, tha, vie |
| License | Unknown (unknown) |
| Homepage | https://entuedu-my.sharepoint.com/:f:/g/personal/bosheng001_e_ntu_edu_sg/EhlGX7Bmi8pMsdZsm6gQr4oBgI4Z7kFznovkuDOdixfLUg?e=kbo7kz |
| HF URL | - |
| Paper URL | https://aclanthology.org/2022.acl-long.115.pdf |

**Comment by dehanalkautsar** [comment]
#self-assign

**Comment by sabilmakbar** [comment]
Hi @dehanalkautsar, may I know the current status of this dataloader creation? Feel free to discuss here if you have any difficulties. Thanks!

**Comment by dehanalkautsar** [comment]
I’ll create all the dataloaders assigned to me this week

**Comment by sabilmakbar** [comment]
Hi @SamuelCahyawijaya, it looks like `V2.0` should lie under the version rather than its dataset name, wdyt?

**Comment by dehanalkautsar** [comment]
I agree with you @sabilmakbar, I've set V2.0 as the dataset's source version in the PR above
